geode-core/src/distributedTest/java/org/apache/geode/internal/cache/tier/sockets/UpdatePropagationDUnitTest.java

```diff
@@ -248,6 +293,20 @@ private void createClientCache(String host, Integer port1, Integer port2) throws
         .addCacheListener(new EventTrackingCacheListener()).create(REGION_NAME);
   }
 
+  private void createClientCache(String host, Integer port1) {
+    ClientCache cache;
```

Comment:
I recommend declaring variables like this on the same line that actually sets it. One less line, and it's easier to read and understand.